resources/asciidoctor/lib/docbook45/converter.rb

```diff
@@ -0,0 +1,171 @@
+# encoding: UTF-8
```

Comment:
This file comes from asciidoctor's source. Well, mostly. I had to do quite a bit of funky things to it to make it work properly in Asciidoctor 2, but I don't believe it'll want to stick to our style or other guidelines. Mostly because we plan to remove it in favor of converting directly to html soon enough anyway.